spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/BraveAutoConfiguration.java

```diff
@@ -166,6 +191,123 @@ BraveHttpClientHandler braveHttpClientHandler(
 			return new BraveHttpClientHandler(httpClientHandler);
 		}
 
+		@Configuration(proxyBeanMethods = false)
+		@ConditionalOnProperty(value = "management.tracing.baggage.enabled", havingValue = "false",
+				matchIfMissing = true)
+		static class BraveNoBaggageConfiguration {
+
+			@Bean
+			@ConditionalOnMissingBean
+			@ConditionalOnProperty(value = "management.tracing.propagation.type", havingValue = "W3C",
+					matchIfMissing = true)
+			Factory w3cPropagationNoBaggageFactory() {
+				return new W3CPropagation(BRAVE_BAGGAGE_MANAGER, List.of()); // TODO: Use
```

Comment:
Correct, I mean if we leave this as it is the components for baggage (i.e. `BraveBaggageManager`) will be called, but no baggage will be propagated cause there's nothing configured. With this change in tracing we are nulling anything baggage related so that will not be called at all